composer.json

```
@@ -0,0 +1,11 @@
+{
```

```suggestion
{
  "name": "Nikschavan/json-ld-breadcrumbs",
  "description": "Adds JSON-LD based breadcrumb schema to your site visible only to the Search Engines.",
```

Let's formalize the composer.json with the project name and description